pkg/config/testdata/tracee-config.yml

```diff
@@ -0,0 +1,57 @@
+kind: TraceeConfig
+apiVersion: tracee.aquasecurity.github.io/v1alpha1
+signatures:
+  - metadata:
```

Comment:
What about other information like mitre technique?

Reply:
We'll add all properties defined in SigMetadata like we do for Rego and Go signatures. I was just lazy to type them in :)